gspread/urls.py

```diff
@@ -19,3 +19,6 @@
 DRIVE_FILES_API_V2_URL = 'https://www.googleapis.com/drive/v2/files'
 DRIVE_FILES_UPLOAD_API_V2_URL = ('https://www.googleapis.com'
                                  '/upload/drive/v2/files')
+
+SPREADSHEET_DRIVE_URL = "https://docs.google.com/spreadsheets/d/%s"
+WORKSHEET_DRIVE_URL = SPREADSHEET_URL + "#gid=%s"
```

Comment:
I noticed that you're using a different template for `WORKSHEET_DRIVE_URL` than in the Sheets UI. Where it's:
```
https://docs.google.com/spreadsheets/d/spreadsheetId/edit#gid=sheetId
```
As described in https://developers.google.com/sheets/api/guides/concepts

What was the reason for dropping the `/edit` part? Looks like the Google server issues a 302 redirect when I try to access the spreadsheet without `/edit` part:

```
$ curl -I https://docs.google.com/spreadsheets/d/my_SeCr4tSh44t-111/#gid=0
HTTP/2 302
content-type: text/html; charset=UTF-8
cache-control: no-cache, no-store, max-age=0, must-revalidate
pragma: no-cache
expires: Mon, 01 Jan 1990 00:00:00 GMT
date: Tue, 18 Feb 2020 11:48:45 GMT
location: https://docs.google.com/spreadsheets/d/my_SeCr4tSh44t-111/edit
```